spec/deprecation_spec.rb

```diff
@@ -102,28 +102,33 @@
     end
 
     context "with bitbucket gems" do
+      let(:user_name) { 'mcorp' }
+      let(:repo_name) { 'flatlab-rails' }
+
       it "warns about removal" do
         allow(Bundler.ui).to receive(:deprecate)
         msg = "The :bitbucket git source is deprecated, and will be removed " \
           "in Bundler 2.0. Add this code to your Gemfile to ensure it " \
           "continues to work:\n    git_source(:bitbucket) do |repo_name|\n  " \
-          "    https://\#{user_name}@bitbucket.org/\#{user_name}/\#{repo_name}" \
+          "    https://#{user_name}@bitbucket.org/#{user_name}/#{repo_name}" \
```

Comment:
OK, looking at the source, the deprecation printed in DSL.rb is wrong -- the interpolation line needs to be in single quotes instead of double quotes